src/pages/student_fees_list/FeesList.tsx

```diff
@@ -0,0 +1,277 @@
+import BackIcon from "../../assets/back-icon.svg";
+import StudentIcon from "../../assets/student-icon.svg";
+import ViewIcon from "../../assets/openEye-icon.png";
+import CoinIcon from "../../assets/coin-icon.svg";
+import type { StudentMonthlyFeeResponse } from "../../app/types/responses/StudentMonthlyFee.type";
+import { useState } from "react";
+import { ConfirmDialog } from "../../components/confirm_dialog/ConfirmDialog";
+import {
+  useGetStudentByIdQuery,
+  useGetStudentMonthlyFeesQuery,
+} from "../../app/services/StudentService";
+import { skipToken } from "@reduxjs/toolkit/query/react";
+import { translateMonth } from "../../utils/TranslateMonths";
+import { formatCurrency } from "../../utils/Formatter";
+import DateFilter from "../../components/date_filter/DateFilter";
+import { toast } from "react-hot-toast";
+import { useUpdateMonthlyFeeMutation } from "../../app/services/MonthlyFeeService";
+import PaymentInfoModal from "../../components/payment_detail/paymentInfo";
+
+function StudentFeesList() {
+  const location = useLocation();
+  const { student } = location.state || {};
+  const { data: studentDetails } = useGetStudentByIdQuery(student.id);
+  const navigate = useNavigate();
+  const [monthFilter, setMonthFilter] = useState("");
+  const [expirationDateFilter, setExpirationDateFilter] = useState<Date | null>(
+    null
+  );
+  const [statusFilter, setStatusFilter] = useState("");
+  const [modalType, setModalType] = useState<"pay" | "details" | null>(null);
+  const [selectedFeeId, setSelectedFeeId] = useState<string | null>(null);
+  const [selectedPaymentId, setSelectedPaymentId] = useState<string | null>(
+    null
+  );
+  const [payMonthlyFee] = useUpdateMonthlyFeeMutation();
+
+  const normalizeDate = (date: Date | string | null) => {
+    if (!date) return undefined;
+
+    const d = new Date(date);
+
+    if (isNaN(d.getTime())) return undefined;
+    return d.toISOString().split("T")[0]; // "YYYY-MM-DD"
+  };
+  const formattedExpirationDate = normalizeDate(expirationDateFilter);
+  const {
+    data: fees,
+    isLoading,
+    isError,
+  } = useGetStudentMonthlyFeesQuery(
+    student?.id
+      ? {
+          studentId: student.id,
+          month: monthFilter || undefined,
+          expirationDate: formattedExpirationDate,
+          status: statusFilter || undefined,
+        }
+      : skipToken
+  );
+
+  const handleOpenPayModal = (feeId: string) => {
+    setSelectedFeeId(feeId);
+    setModalType("pay");
+  };
+  const handleOpenPaymentInfoModal = (paymentId?: string) => {
+    setSelectedPaymentId(paymentId!);
+    setModalType("details");
+  };
+  const handleConfirmPay = async () => {
+    if (!selectedFeeId) return;
+    payMonthlyFee({
+      feeId: selectedFeeId,
+      studentId: student.id,
+    })
+      .unwrap()
+      .then(() => {
+        toast.success("Pago realizado correctamente");
+      })
+      .catch(() => {
+        toast.error("Ocurrió un error al procesar el pago");
+      })
```

Comment:
Los mensajes los modificaria


```suggestion
      .then(() => {
        toast.success("Cuota registrada como pagada correctamente");
      })
      .catch(() => {
        toast.error("Ocurrió un error registrar la cuota como pagada");
      })
```

Esto porque nosotros no procesamos ni hacemos un pago realmente, sino que solamente marcamos la cuota como pagada